code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -24,6 +24,47 @@
 	QDEL_NULL(modifier)
 	return ..()
 
+/datum/reagent/epinephrine
+	name = "Epinephrine"
+	id = "epinephrine"
+	description = "Epinephrine, also known as adrenaline, is a super strength stimulant and painkiller intended to keep a patient alive while in critical health."
```

Comment:
These descriptions show up in the chemmaster when the analyse button is used. Phrases like "critical health" are a bit odd to use in-character, and bring up memories of low-RP doctors shouting "HE'S IN CRIT", etc. Maybe "critical condition"?